backend/blueprint/api/controllers/loginUser.py

```diff
@@ -21,8 +21,8 @@ def login():
                 str(e),
                 'Unable to get json'))
 
-    if data and data['username']:
-        user = User.getUser(username=data['username'])
+    if 'name' in data.keys():
+        user = User.getUser(data['name'])
```

Comment:
@yashLadha a little confusion here! The getUser method filters user by his/her username. But here you are sending name. In register module, name is used for display name, it is present in firebase schema but not in our database service.